frontend/ui/BUILD

```diff
@@ -52,7 +52,7 @@ web_app(
 pkg_tar(
     name = "bundle_tar",
     srcs = [":bundle"],
-    package_dir = "/usr/share/nginx/html/",
```

Comment:
It's a bit weird that the ui package has to be aware of this change. I would prefer that we rewrite the path by stripping the prefix [in traefik](https://doc.traefik.io/traefik/middlewares/stripprefix/)